NitroxModel/Packets/CoffeeMachineUse.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using NitroxModel.DataStructures;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class CoffeeMachineUse : Packet
+    {
```

Comment:
Can coffee machine "stores" coffee, or it's just a one-shot use and it gives coffee to near player ?